### ISSUE TITLE
Can you make `?inline` configurable? 

### ISSUE THREAD

**Issue by gkatsanos** [gammacommunications/vue-svg-loader-2#58]
We have a component library that's consumed by both webpack & vite applications, we'd like to use the vite svg loader in some apps but `?inline` is not supported. ( we need `?component` )
we'd love it if you can make inline configurable.

**Comment by ndrscodes** [comment]
i'll have a look at it ASAP! could you explain the feature a little more so i can find a fix for it faster? :)

**Comment by gkatsanos** [comment]
So any import url with `?inline` at the end of the url inlines the svg markup into the HTML. We were wondering if `inline` can be configurable.

**Comment by ndrscodes** [comment]
hmmm, i'm still not quite sure i understand exactly what should happen when passing `?inline` (i guess as a url parameter?) could you show me an example of the `inline` call and the corresponding html you expect to be generated?

**Comment by gkatsanos** [comment]
could this be something coming from another dependency of this package? webpack related? not sure anymore.. I thought its a feature of this extension.

**Comment by ndrscodes** [comment]
since we are controlling the way the html is rendered through webpack, it's more like a feature request. I will gladly find a solution for this one :) But i'll need an explaination of what exactly the parameter should do, ie. how the template should look like once a svg is loaded using this `inline` parameter, and if possible an example of where `inline` is used

**Comment by gkatsanos** [comment]
Sorry I think I may have not explained myself correctly. 
We're not building a new feature - we're taking the existing feature and customizing the url parameter (now: `inline`) to be passed by the user in the plugin configuration..

**Comment by ndrscodes** [comment]
Where did you find docs about an `inline` parameter? I'm trying to find out what it should do. From the original docs, it looks like it simply causes the vue-svg-loader to be used. If it's not specified, a different loader would be used instead. Am i seeing this correctly?

**Comment by gkatsanos** [comment]
honestly I didn't read docs about it as it was already there, but I believe what you found was accurate!

**Comment by ndrscodes** [comment]
i found it in the original vue-svg-loader docs: https://vue-svg-loader.js.org/faq.html#how-to-use-both-inline-and-external-svgs

**Comment by gkatsanos** [comment]
yep, thats the one. Do you think this can be configured?

**Comment by ndrscodes** [comment]
i don't think this is something we can solve on our part. you'd have to configure it in your own webpack/vue config. If it works, please let me know so i can add it to our own readme!

**Comment by ndrscodes** [comment]
@gkatsanos did you habe any luck when configuring webpack? Is there something we should include in the readme to help people with this in the future?

**Comment by gkatsanos** [comment]
We are going to make the parameter configurable on the vite loader side!

**Comment by ndrscodes** [comment]
@gkatsanos let me know how you did it so i can add it to our own ReadMe :)

**Comment by ndrscodes** [comment]
Hey @gkatsanos, did you manage to configure the ˋinlineˋ Parameter? If so, would you like us to put it somewhere in the README?

**Comment by gkatsanos** [comment]
@ndrscodes we have a PR to add an alias to our side so you dont need to do much. https://github.com/jpkleemans/vite-svg-loader/pull/107

**Comment by ndrscodes** [comment]
in that case i'll close the issue for now :)
